plugins/serializers/wavefront/wavefront.go

```diff
@@ -50,16 +58,17 @@ func NewSerializer(prefix string, useStrict bool, sourceOverride []string) (*Wav
 
 // Serialize : Serialize based on Wavefront format
 func (s *WavefrontSerializer) Serialize(m telegraf.Metric) ([]byte, error) {
-	out := []byte{}
-	metricSeparator := "."
+	const metricSeparator = "."
+	var out []byte
+	buf := pbFree.Get().(*buffer)
```

Comment:
It probably makes sense to just create a single bytes.Buffer and then pass this into the various functions, the `out` slice would be removed and then there is no reason to append.  I don't see a reason we need both of these.

Reply:
Agreed, will change.

Reply:
Expect something from me tomorrow or Friday as I'm pretty short on bandwidth right now.